drgn/helpers/linux/pid.py

```diff
@@ -123,3 +125,39 @@ def for_each_task_in_group(
     ):
         if other != task or include_self:
             yield other
+
+
+def is_kthread(t: Object) -> bool:
```

Comment:
I think `task_is_kthread()` would be a better name for this one, `task` would be a better name for the argument, and the helper would make more sense in `drgn.helpers.linux.kthread`.